test/suites/backup.sh

```diff
@@ -680,11 +686,11 @@ test_backup_volume_export_with_project() {
   rmdir "${LXD_DIR}/optimized"
   rmdir "${LXD_DIR}/non-optimized"
 
-  if storage_backend_available "ceph" && [ -n "${LXD_CEPH_CEPHFS:-}" ]; then
+  if "${use_cephfs}"; then
```

Comment:
Based on my comment, this should be done in `test_backup_volume_export` after the tests have run.